tensornetwork/network.py

```diff
@@ -76,6 +77,7 @@ def _new_node_name(self, name: Optional[Text]) -> Text:
   def dtype(self) -> Type[np.number]:
     return self.backend.dtype
 
+  # pylint: disable=redefined-outer-name
```

Comment:
Why did you disable this?

Reply:
that's an artifact i forgot to remove. It's gone